### ISSUE TITLE
SyncTeX works tex->pdf, but not reverse

### ISSUE THREAD

**Issue by rickevertsz** [James-Yu/Atom-LaTeX#169]
After running atom-latex:preview to preview the file, atom-latex:synctex takes me from a .tex file to the right place in the pdf, but if I click on text in the pdf pane and invoke atom-latex:synctex (via Ctrl-L Ctrl-S), it brings up a Save Dialog.

I'm wondering if this is user error, but I've run out of ideas.

**Comment by ashthespy** [comment]
Have you tried `SyncTeX from here` right click menu? That should determine if it' a key binding issue, or an actual syncTeX issue. 
You can always change the keybinding if you find the defaults not suitable for you :-) 

PS: I like to think of it more as `ctrl` - `L` - `S` i.e holding down `ctrl` and hitting `l` `s` ;-)

**Comment by rickevertsz** [comment]
Ok, I missed that `Cmd-click` is the way to go. I overlooked that in the documentation because I jumped straight to the keybinding table in my eagerness to edit LaTeX in Atom. I actually tried `Cmd-click` on my machine at work before opening this issue (because that's how it is invoked in TeXShop), but it did nothing; works fine on my laptop at home right now, however.

So, it's user error of a sort :) Had I read the documentation thoroughly, I would have persevered with `Cmd-click` on my machine at work. Perhaps a restart of Atom would have fixed it.

So, `Ctrl-L Ctrl-S` is a chord?! Having grown up on Emacs, I can't help but see it as two keystrokes ;-)

Thanks very much for your help, I'm fairly new to Atom and this `atom-latex` package looks *really* nice.
